bazel/com_github_google_flatbuffers.patch

```diff
@@ -0,0 +1,13 @@
+diff -ruN a/include/flatbuffers/base.h b/include/flatbuffers/base.h
+--- a/include/flatbuffers/base.h	2022-09-30 18:32:00.076755650 +0200
++++ b/include/flatbuffers/base.h	2022-09-30 18:32:00.103755894 +0200
+@@ -258,6 +258,9 @@
+   #endif
+ #endif // !FLATBUFFERS_HAS_NEW_STRTOD
+ 
++// The following define added to fix OSSM-1931
++#define FLATBUFFERS_LOCALE_INDEPENDENT 0
```

Comment:
This can be passed as a command line argument in bazel, no?

Reply:
Yes I think so

Reply:
The working way I've found to do that is using the flag_set of Bazel in maistra/vendor/emsdk/emscripten_toolchain/crosstool.bzl:

flag_set(
            actions = all_compile_actions,
            flags = ['-DFLATBUFFERS_LOCALE_INDEPENDENT=0'],
        ),

Reply:
I was think something like https://github.com/maistra/envoy/blob/maistra-2.2/maistra/bazelrc#L6

Reply:
Yes, I had tried it, for example, in BUILD file:

time bazel_build --copt -DFLATBUFFERS_LOCALE_INDEPENDENT=0 //extensions:stats.wasm

but didn't work

Reply:
No success even in:
COMMON_FLAGS="\
    --copt -DFLATBUFFERS_LOCALE_INDEPENDENT=0 \
    --config=release \
    --config=${ARCH} \
"